File: Homework/Homework015/Task2.py
# Определить индексы элементов массива (списка), значения 
# которых принадлежат заданному диапазону (т.е. не меньше 
# заданного минимума и не больше заданного максимума)

from random import randint as rnd

min = 10
max = 12

list = []
for i in range(20):
    list.append(rnd(1, 100))

list_of_index = []
for i in range(len(list)):
    if min <= list[i] <= max:
        list_of_index.append(i)

print(list)
print(list_of_index)

File: Homework/Homework019/Task2.java
package Homework019;

import java.util.ArrayList;
import java.util.Collections;

/**
 * Задан целочисленный список ArrayList. Найти минимальное, максимальное и
 * среднее арифметическое из этого списка.
 */
public class Task2 {

    public static void main(String[] args) {

        ArrayList<Integer> array = new ArrayList<>();

        for (int i = 0; i < 20; i++) {
            Double x = Math.random() * 10;
            Integer number = x.intValue();
            array.add(number);
        }
        System.out.println(array);

        double sum = 0;
        for (int i = 0; i < array.size(); i++) {
            sum += array.get(i);
        }
        double average = sum / array.size();

        System.out.printf("Max value: %d, min value: %d, average: %.2f", (Collections.max(array)),
                (Collections.min(array)), average);

    }
}